infrastructure/builder/src/release/index.js

```diff
@@ -66,7 +71,12 @@ class Release {
     console.log(`Release version: ${context['release-version']}`);
     console.log(`Release docker image: ${context['monoimage-docker-image']}`);
     if (!this.cmdOptions.push) {
-      console.log('  NOTE: image and git commit + tag not pushed (use --push)');
+      console.log('NOTE: image and git commit + tag not pushed (use --push)');
+    } else {
+      console.log(`GitHub release: ${context['github-release']}`);
+      console.log('** YOUR NEXT STEPS **');
+      console.log(' * run `npm publish` in clients/client');
```

Comment:
Also client-web?

Reply:
Yes!  Will add.

It shouldn't conflict -- it's just formalizing some of the stuff that was done by hand.